specification/network/resource-manager/Microsoft.Network/2017-09-01/examples/ApplicationSecurityGroupCreate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2017-09-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "applicationSecurityGroupName": "test-asg",
+    "location": "westus",
+    "parameters": {
+      "properties": {
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "test-asg",
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationSecurityGroups/test-asg",
+        "type": "Microsoft.Network/applicationSecurityGroups",
+        "location": "westus",
+        "properties": {
+          "provisioningState": "Succeeded"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "name": "test-asg",
+        "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/applicationSecurityGroups/test-asg",
+        "type": "Microsoft.Network/applicationSecurityGroups",
+        "location": "westus",
+        "properties": {
+          "provisioningState": "Succeeded"
```

Comment:
If this is long running then shouldn't the provisioningState be Creating or some other value that states it is non terminal.

Reply:
It could be either Succeeded or Updating. I'll leave it as it is for this one.